docs/dashboard/app-templates-gallery.md

```diff
@@ -0,0 +1,70 @@
+---
+title: App Templates Gallery (preview)
+---
+
+:::info
+
+<!-- todo: change -->
+
+This feature was introduced in **Saleor 3.8**.
+:::
+
+:::caution
+This feature is currently in **Feature Preview**. This part of Saleor is not complete
+and subject to change but is available to experiment and provide feedback.
+:::
+
+## What is App Templates Gallery?
+
+Apps are the centerpiece of the [extending Saleor model](../developer/extending/apps/key-concepts.mdx). They are the preferred way of building custom logic on top of Saleor.
+
+Saleor Cloud Marketplace is your headquarters for managing and installing apps.
+
+:::note
+You can find Saleor Cloud Marketplace by going to _Apps_ -> _Marketplace_ in your Saleor Dashboard.
+:::
+
+There are two types of software you can find in Saleor Cloud Marketplace: **templates** and **apps**.
```

Comment:
Problably we should change that. There is, I think:
- App Template Gallery
- AND Cloud Marketplace (in progress)

So maybe we can change to
"There are two types of apps you can use in Saleor"